app/presenters/reservation_calendar.rb

```diff
@@ -0,0 +1,50 @@
+class ReservationCalendar < SimpleDelegator
+
+  attr_accessor :url, :ical
+
+  def self.to_calendar(reservation, url = nil)
+    new(reservation, url).to_ical
+  end
+
+  def initialize(reservation, url = nil)
```

Comment:
Kept the URL to keep the wrapper logic. But making it a keyword is easy enough, and I think it's fine to remove the class method